muffin_man.gemspec

```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |spec|
 
   spec.add_development_dependency "rspec", "~> 3.2"
   spec.add_development_dependency 'webmock', '~> 2.1'
-  spec.add_development_dependency 'byebug'
+  spec.add_development_dependency 'byebug', '~> 11.1.3'
```

Comment:
Added version to fix Rubocop action failure
Ref:https://github.com/patterninc/muffin_man/actions/runs/4282074516/jobs/7455882747#step:6:15